.travis.yml

```diff
@@ -2,43 +2,25 @@ language: node_js
 sudo: false
 dist: trusty
 before_script:
-  - node script/create-test-tables.js pg://postgres@127.0.0.1:5432/postgres
+  - node script/create-test-tables.js pg://postgres@127.0.0.1:$PGPORT/postgres
 env:
   - CC=clang CXX=clang++ npm_config_clang=1 PGUSER=postgres PGDATABASE=postgres
 
-node_js: "6"
-addons:
-  postgresql: "9.6"
+env:
```

Comment:
Didn't know that travis could handle these environment combinations:

* PG 9.5 - node6
* PG 9.5 - node8
* PG 10 - node6
* PG 10 - node8

Great!

Reply:
If it weren't because PG10 is not fully integrated the script could be simpler (no manual install). For example:
```
postgresql:
  - "9.5"
  - "9.6"

node_js:
  - "6"
  - "8"
```
Will create a matrix of 4 configurations automatically with those packages setup properly.